src/pages/RequestCallPage.js

```diff
@@ -174,10 +198,37 @@ class RequestCallPage extends Component {
         return {firstName, lastName};
     }
 
+    getWaitTimeMessageKey(minutes) {
+        if (!minutes) {
+            return 'requestCallPage.waitTime.calculating';
+        }
+        if (minutes > 300) {
+            // The wait time is longer than 5 hours, so just say that.
+            return 'requestCallPage.waitTime.fiveHoursPlus';
+        }
+
+        if (minutes > 60) {
+            // The wait time is between 1 and 5 hours, so lets convert to hours and minutes.
+            return 'requestCallPage.waitTime.hoursAndMinutes';
+        }
+
+        // The wait time is less than an hour so just give minutes.
+        return 'requestCallPage.waitTime.minutes';
+    }
+
+    getWaitTimeMessage() {
+        const waitTimeKey = this.getWaitTimeMessageKey(this.props.inboxCallUserWaitTime);
+        return `${this.props.translate(waitTimeKey)} ${this.props.translate('requestCallPage.waitTime.guides')}`;
```

Comment:
Are you ever passing the number of minutes to the translation? And have you tested this by having the API call return different numbers to see if it works for the different length scenarios? 

You can try it by setting this to return different numbers: https://github.com/Expensify/Web-Expensify/blob/main/api.php#L392

Reply:
Ah right I missed that. Thank you for showing me how to easily test with different numbers of minutes. I got it working now.